docs/set-up-environment/windows/index.mdx

```diff
@@ -43,7 +43,7 @@ PowerShell Core, a complete rewriting of PowerShell to make it easier to use and
 
 ### Winget
 
-Microsoft is a great company, but seemingly incapable at advertising actual _useful_ tools. Unbeknownst to most, like Mac, Windows has its own package manager, _[Winget](https://winget.run)_. Using this'll make your life much easier when it comes to installing developer tools.
+Microsoft is a great company, but seemingly incapable at advertising actual _useful_ tools. Unbeknownst to most, like Mac, Windows has its own package manager, Winget. Using this'll make your life much easier when it comes to installing developer tools.

```

Comment:
@ariwo17 just asking because I might not have been aware of the fact that there was confusion on the hyperlink design: was it difficult to read? or did people just not use the links? Do you intend on adding a list of links at the bottom instead?

I suppose that I'm trying to understand the solution to the problem here.